Rakefile

```diff
@@ -1 +1,84 @@
+#  Copyright 2015 Puppet Community
```

Comment:
> A trademark is any word, name, symbol, or design, or any combination thereof, used in commerce to identify and distinguish the goods of one manufacturer or seller from those of another and to indicate the source of the goods.

A registered trademark is not the same thing as a legal entity. We don't need to be a trademark.

As far as copyright assignment goes, I'll need to have a look at how that works with APLv2 licenses.